manim/mobject/svg/text_mobject.py

```diff
@@ -339,122 +318,213 @@ def text2svg(self):
+            start_i += 2 * rect_counts
+        text_backgrounds.set_color(self.background_color)
+
+        return text_backgrounds
+
+    def text2svg1(self):
+        # anti-aliasing
+        size = self.size * 10
+        line_spacing = self.line_spacing * 10
+
+        if self.font == "":
+            if NOT_SETTING_FONT_MSG != "":
+                logger.warning(NOT_SETTING_FONT_MSG)
+        dir_name = consts.TEXT_DIR
+        hash_name = self.text2hash()
+        file_name = os.path.join(dir_name, hash_name) + ".svg"
+        # if os.path.exists(file_name):
+        # return file_name
+
+        surface = cairo.SVGSurface(file_name, 600, 400)
+        context = cairo.Context(surface)
+        context.set_font_size(size)
+        context.move_to(START_X, START_Y)
 
+        settings = self.text2settings()
+        offset_x = 0
+        last_line_num = 0
+        for setting in settings:
+            font = setting.font
+            slant = self.str2slant(setting.slant)
+            weight = self.str2weight(setting.weight)
+            text = self.text[setting.start : setting.end].replace("\n", " ")
+            context.select_font_face(font, slant, weight)
+            if setting.line_num != last_line_num:
+                offset_x = 0
+                last_line_num = setting.line_num
+            tempx = START_X + offset_x
+            tempy = START_Y + line_spacing * setting.line_num
+            char_offset_x = 0
+            char_height = tempy - size / 2 - (line_spacing - size)
+            # context.set_font_matrix(cairo.Matrix(13, 0, 0, 13, 0, 0))
+            # print(context.get_font_matrix(),size)
+            # if (context.text_extents("99").width-context.text_extents("11").width)/size > 0.13:
+            # for char_index in range(text.__len__()):
+
+            # else:
+
+            print(context.text_extents("  ").width)
+            """
+            for char_index in range(text.__len__()):
+                (x, y, width, height, dx, dy) = context.text_extents("a" + text[char_index] + "a")
+                (_, _, widtha, _, _, _) = context.text_extents("a")
+                width -= 2 * widtha
+                context.rectangle(tempx + char_offset_x, char_height, width, size)
+                context.fill()
+                char_offset_x += width
+            """
+            context.move_to(tempx, tempy)
+            context.show_text(text)
+            offset_x += context.text_extents(text)[4]
+        surface.finish()
         return file_name
 
 
-class TextWithFixHeight(Text):
-    def __init__(self, text, **kwargs):
-        Text.__init__(self, text, **kwargs)
-        max_height = Text("(gyt{[/QW", **kwargs).get_height()
-        rectangle = Rectangle(
-            width=0, height=max_height, fill_opacity=0, stroke_opacity=0, stroke_width=0
-        )
-        self.submobjects.append(rectangle)
+"""
+paragraph paragraph.chars is VGroup() of each lines and each line is VGroup() of that line's characters 
```

Comment:
Question : I don't think this is in the right format. What do you think @PgBiel ?

Reply:
my English is weak 😅

Reply:
No ahah your english is not the issue. The issue is like I don't think this fits our guidelines for docstring.

Reply:
Yeah this should be in the class's docstring, while attrs should be in the Attributes section. Please apply this to other classes as well